.github/workflows/build-os-image.yml

```diff
@@ -581,18 +581,163 @@ jobs:
+                .measurements.3.expected = "3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969" |
+                .measurements.4.warnOnly = false |
+                .measurements.5.warnOnly = true |
+                .measurements.5.expected = "b731e1a1c70e65e43a4ed54d668b7e585163d9c42f1ed4096174451ff0ba62f2" |
+                .measurements.7.warnOnly = true |
+                .measurements.7.expected = "346547a8ce5957af27e552427d6b9e6d9cb502f0156e9155380451eea1b3f0ed" |
+                .measurements.8.warnOnly = false |
+                .measurements.9.warnOnly = false |
+                .measurements.11.warnOnly = false |
+                .measurements.12.warnOnly = false |
+                .measurements.13.warnOnly = false |
+                .measurements.14.warnOnly = true |
+                .measurements.14.expected = "d7c4cc7ff7933022f013e03bdee875b91720b5b86cf1753cad830f95e791926f" |
+                .measurements.15.warnOnly = false' \
+                "${OUTFLAGS}" "${PCRFILE}"
+            ;;
+            gcp)
+              yq e '.csp = "GCP" |
+                .image = "${{ needs.build-settings.outputs.imageNameShort }}" |
+                .measurements.0.warnOnly = false |
+                .measurements.0.expected = "0f35c214608d93c7a6e68ae7359b4a8be5a0e99eea9107ece427c4dea4e439cf" |
+                .measurements.1.warnOnly = true |
+                .measurements.1.expected = "745f2fb4235e4647aa0ad5ace781cd929eb68c28870e7dd5d1a1535854325e56" |
+                .measurements.2.warnOnly = true |
+                .measurements.2.expected = "3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969" |
+                .measurements.3.warnOnly = true |
+                .measurements.3.expected = "3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969" |
+                .measurements.4.warnOnly = false |
+                .measurements.5.warnOnly = true |
+                .measurements.5.expected = "9d5f8c2e0464c69552e593be32a4b089f7e77f4ed12d60cdcd94120b6ccaccf8" |
+                .measurements.6.warnOnly = true |
+                .measurements.6.expected = "3d458cfe55cc03ea1f443f1562beec8df51c75e14a9fcf9a7234a13f198e7969" |
+                .measurements.7.warnOnly = true |
+                .measurements.7.expected = "b1e9b305325c51b93da58cbf7f92512d8eebfa01143e4d8844e40e062e9b6cd5" |
+                .measurements.8.warnOnly = false |
+                .measurements.9.warnOnly = false |
+                .measurements.10.warnOnly = true |
+                .measurements.10.expected = "7f96fbc55e1d2a0de46e5d44658c06ef102d1198703efa69f2ea6b5aa1c9a176" |
+                .measurements.11.warnOnly = false |
+                .measurements.12.warnOnly = false |
+                .measurements.13.warnOnly = false |
+                .measurements.14.warnOnly = true |
+                .measurements.14.expected = "d7c4cc7ff7933022f013e03bdee875b91720b5b86cf1753cad830f95e791926f" |
```

Comment:
PCR 14 is coming from our bootchain IIRC.

Reply:
No, we use [4,8,9,11,12,13,15].
Not 100% sure where 14 comes from, but we don't enforce it currently.
Maybe @malt3 knows if this is some part of systemd?

Reply:
PCR 14 comes from shim:
https://github.com/rhboot/shim/blob/main/README.tpm

So part of our bootchain / secure boot.